iroha-lib/README.md

```diff
@@ -1,16 +1,118 @@
-# C++ library for Hyperledger Iroha
+# Hyperledger Iroha C++ library
 
 
-This is a source code for HL Iroha C++ library.
+### Current version of the library was tested and compatible with Iroha v2.0.0-pre.rc.6
 
-Currently, latest HL Iroha 1.5 release (`hyperledger/iroha:latest` Docker image) is supported.
+The library was created to provide a convenient interface for C++ applications to communicate with [Iroha](https://github.com/hyperledger/iroha) blockchain including sending transactions/query, streaming transaction statuses and block commits.
 
 
-### Installation
+# iroha-lib
 
-```bash
-mkdir build && cd build && cmake .. && make 
+Client library of [Iroha](https://github.com/hyperledger/iroha) written completely in modern C++.
+Currently, the latest HL Iroha 1.5 release (`hyperledger/iroha:latest` Docker image) is supported.
+
+
+## Installation
+
+Please go through steps below to run the project:
+
+1. Follow instructions from [Iroha documentation](https://iroha.readthedocs.io/en/main/getting_started/) to setup and run Iroha peer in [Docker](https://www.docker.com) container.
+
+2. Clone current repository.
+
+3. cd `examples` directory. Build and Run examples ```mkdir build && cd build && cmake .. && make```.
```

Comment:
I think it will not work that easily, because to run CMake You need to provide:
```
-DCMAKE_TOOLCHAIN_FILE=$PWD/vcpkg-build/scripts/buildsystems/vcpkg.cmake
```
______________
I managed to run example by calling commands (according to [build instruction for Linux](https://iroha.readthedocs.io/en/develop/build/index.html#linux) in documentation):
```
git clone -b develop https://github.com/hyperledger/iroha.git
cd iroha/
./vcpkg/build_iroha_deps.sh $PWD/vcpkg-build
cmake -B build -DCMAKE_TOOLCHAIN_FILE=$PWD/vcpkg-build/scripts/buildsystems/vcpkg.cmake . -DCMAKE_BUILD_TYPE=RELEASE   -GNinja -DUSE_BURROW=OFF -DUSE_URSA=OFF -DTESTING=OFF -DPACKAGE_DEB=OFF
cmake --build ./build --target batch_example
cmake --build ./build --target batch_example
./build/bin/batch_example 
```

Reply:
Ok. I simplified description.